### PR TITLE
Write Hello World 

### PR DESCRIPTION
**Pull request by chryssavarna**
Ask for User Input

### CODE REVIEW

File: HelloWorld.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Print an introduction
+print("Welcome to Python")
+#Ask for input
+
+text = input("Would you like to print Hello World? Y(yes) or N(No)")
```
Comment (Popcore) [inline HelloWorld.py]: `input` works well with python 3 but not really with python 2 - where `raw_input` is the way to take use inputs. I suggest to specify in you README file that this code requires python 3 in order to run.

File: HelloWorld.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+#Check your status
+if text == "Y":
+    print("Hello World")
+else:
+    pass
```
Comment (Popcore) [inline HelloWorld.py]: I'm being unnecessarily strict here:  the message tells the user to either type "Y" or "N" but you are only handling the first case. Maybe add a message when a user press "N" or other keys?

**Review by Popcore — CHANGES_REQUESTED** [review]

**Comment by chryssavarna** [comment]
@Popcore how does this look?

**Review by Popcore — APPROVED** [review]
Looks good. 

A possible improvement could be to make your functions a bit more flexible so that they can accept parameters and allow for reusability. I'm specifically thinking about the `AskQuestion` function. 
But the script does its job and works as expected so if you are happy with it go ahead and merge.

**Comment by Popcore** [comment]
Yey 🎉

btw new year new github account. I am [here](https://github.com/ocramh) now

**Pull request merged**